tsconfig.json

```diff
@@ -40,7 +40,8 @@
   },
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
-    "strictInjectionParameters": true
+    "strictInjectionParameters": true,
+    "enableIvy": false
```

Comment:
so are we not enabling Ivy in ids-ep-ng 7?

Reply:
Not for the demo app, as there is (was) an issue with SVG icons.  See the CHANGELOG for details.